_posts/blog/2017-07-19-Actions-for-android-devs.md

```diff
@@ -8,19 +8,18 @@ tags: [Google Assistant, Actions on Google, conversational interfaces, Android,
 
 
-For the uninitiated, Actions on Google are third party "Actions" that can be built using Google's APIs.  Actions extend what the Google Assistant can do by letting user talk to them.  They can be accessed with trigger words such as 'Ok Google, talk to Ticketmaster', or 'Ok Google, ask ticketmaster find Yankee's games this weekend' (Ticketmaster Action coming soon!)
+For the uninitiated, Actions on Google are third-party apps that can be built using Google's APIs.  Actions extend what the Google Assistant can do by letting user talk to them.  They can be accessed with trigger words such as 'Ok Google, talk to Ticketmaster', or 'Ok Google, ask ticketmaster find Yankee's games this weekend'. (Ticketmaster Action coming soon!)
 
 <img src="https://storage.googleapis.com/banjotms.appspot.com/Screenshot_20170718-103901.png" width="300"> 
 
-At Ticketmaster Mobile Studio, we decided to use the languages and dev environment we are most familiar with - Kotlin/Java, IntelliJ, & Gradle.  This allows us to use the language and tools we are familiar with, and to reuse some code with Android apps.  However, the documentation and examples do not provide any Java/Kotlin.  Google provides a javascript SDK for building webhooks, however so far there is not an official Java SDK.  At the end of the day this is just json in and json out, so of course we can use any language we choose.
+At Ticketmaster Mobile Studio, we decided to use the languages and dev environment we are most familiar with - Kotlin/Java, IntelliJ, & Gradle.  This allows us to use the language and tools we are comfortable with, and to reuse some code with Android apps.  Unfortunately, the documentation and examples are only in nodejsdo not provide any Java/Kotlin.
 
-
-Looking through the json specs and creating response objects to match is pretty tedious, so we have decided to open source an [unofficial Kotlin/Java Actions-on-google SDK](https://github.com/TicketmasterMobileStudio/actions-on-google-kotlin).  This is an early release and will we be adding functionality, with a goal of matching the official SDK.  Currently, all the conversation components are supported.  Using the Kotlin SDK you can get up and running in no time.  And since Kotlin has great interopt with Java, you can use it from Java as well.
+Looking through the json specs and creating response objects to match is pretty tedious, so we have decided to open source an [unofficial Kotlin/Java Actions-on-google SDK](https://github.com/TicketmasterMobileStudio/actions-on-google-kotlin).  This is an early release and will we be adding functionality, with a goal of matching the official SDK.  Currently, all the conversation components are supported.  Using the Kotlin SDK you can get up and running in no time.  And since Kotlin has great interop with Java, you can use it from Java as well.
```

Comment:
"excitment"

"for a number of devices" --> "for multiple devices"

"For the uninitiated," --> I'd remove this

"can do by letting user talk" --> "can do by letting the user talk"

"Ok Google, ask ticketmaster find" --> capitalize Ticketmaster, and make it "to find"

"Yankee's games" --> no apostrophe

"we decided to use the languages and dev environment we are most familiar with - [...] This allows us to use the language and tools we are comfortable with" --> This is kind of repetitive. Also states the obvious. Maybe "leverage our expertise" or something like that?